packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-state.ts

```diff
@@ -333,9 +333,8 @@ const getLineCoords = ({
     step,
 }: getLineCoordsArg): PairOfPoints[] => {
     if (graph.coords) {
-        return [graph.coords];
+        return graph.coords;
```

Comment:
This should always be the right type (`PairOfPoints[]`) now, so we don't need to add it to another array.